roles/tests-integration/tests/pool_integration.rs

```diff
@@ -94,3 +95,61 @@ async fn test_sniffer_interrupter() {
     assert_common_message!(&sniffer.next_message_from_downstream(), SetupConnection);
     assert_common_message!(&sniffer.next_message_from_upstream(), SetupConnectionError);
 }
+
+// covers
+// https://github.com/stratum-mining/stratum/blob/main/test/message-generator/test/translation-proxy/translation-proxy.json
+#[tokio::test]
+async fn translation_proxy() {
+    let pool_jdc_sniffer_addr = common::get_available_address();
+    let tp_addr = common::get_available_address();
+    let pool_addr = common::get_available_address();
+
+    let pool_jdc_sniffer = common::start_sniffer(
+        "0".to_string(),
+        pool_jdc_sniffer_addr,
+        pool_addr,
+        false,
+        None,
+    )
+    .await;
+    let _tp = common::start_template_provider(tp_addr.port()).await;
+    let _pool_1 = common::start_pool(Some(pool_addr), Some(tp_addr)).await;
+    let jds_addr = common::start_jds(tp_addr).await;
+    let jdc_addr = common::start_jdc(pool_jdc_sniffer_addr, tp_addr, jds_addr).await;
```

Comment:
why do we have JDC + JDS on this test?

this should be JD-less, tProxy should simply connect to Pool